scripts/Makefile.am

```diff
@@ -158,6 +158,7 @@ EXTRA_DIST =			\
 	update_submodules.sh	\
 	mcs.in				\
 	csc.in				\
+	vbc.in				\
```

Comment:
We need `vbc` in bin_SCRIPTS as well, and a target to create it. Look for the csc target at L208 for an example.